### DISCUSSION TITLE
第三方api调用问题

### DISCUSSION THREAD

**Discussion by cyxer000** [lobehub/lobehub#4337] [General Question]
{
  "error": {
    "status": 400,
    "headers": {
      "access-control-allow-origin": "https://****.org.cn",
      "content-length": "153",
      "content-type": "application/json; charset=utf-8",
      "date": "Sat, 12 Oct 2024 09:30:37 GMT"
    },
    "error": {
      "type": "error",
      "error": {
        "type": "invalid_request_error",
        "message": "The model `claude-3-haiku-20240307` does not exist or you do not have access to it."
      }
    }
  },
  "endpoint": "https://api.de****ks.ai/",
  "provider": "anthropic"
}

**Comment by arvinxx** [comment]
这不是写了模型不存在或者没资源么，找下你的供应商
